metadata-ingestion/tests/unit/sdk/test_mce_builder.py

```diff
@@ -8,6 +8,9 @@
     MetadataChangeEventClass,
     OwnershipClass,
 )
+from datahub.utilities.urn_encoder import RESERVED_CHARS_EXTENDED
+
+_RESERVED_CHARS_STRING = "".join(sorted(list(RESERVED_CHARS_EXTENDED)))
```

Comment:
this can probably just be a hardcoded string. if the expected urns are hardcoded, this should be too. 

ultimately as long as something is getting encoded, we know the encode_string call is happening and so we're fine on testing